action.yaml

```diff
@@ -104,5 +101,7 @@ runs:
         tags: ${{ steps.meta.outputs.tags }}
         build-args: ${{ inputs.build-args }}
         labels: ${{ inputs.labels }}
-        cache-from: type=gha
-        cache-to: type=gha,mode=max
+        cache-from: ${{ fromJSON(inputs.no-cache) == true && '' || 'type=gha' }}
```

Comment:
The docs say "It is important to note that the first value after the && must be truthy. Otherwise, the value after the || will always be returned."
Even if this works, maybe it would be safer to invert the test?